SalesforceCommerceCloud/pwa-kit
# W-12627155 Fix product tile img alt text for a11y

# Description
Alt text in product tile img is redundant since it repeats the product name which can cause confusion for visually impaired individuals. The img can be considered decorative for the tile, we can just leave it empty.



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- Run retail app locally
- Open PLP
- Inspect Product Tile
- Make sure the alt text on img in product tile is empty


# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
